Index: data/mymeds.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\"patient\": [\n    {\n        \"birthday\": {\n            \"month\": 7,\n            \"year\": 1980,\n            \"day\": 31\n        },\n        \"drugs\": [{\n            \"instructions\": \"eat\",\n            \"dose\": \"50g\",\n            \"name\": \"gillyweed\",\n            \"indication\": \"breathing underwater\"\n        }],\n        \"name\": \"harry\"\n    },\n    {\n        \"birthday\": {\n            \"month\": 9,\n            \"year\": 1979,\n            \"day\": 19\n        },\n        \"drugs\": [{\n            \"instructions\": \"as needed\",\n            \"dose\": \"1 cup\",\n            \"name\": \"polyjuice potion\",\n            \"indication\": \"change into someone else\"\n        }],\n        \"name\": \"hermione\"\n    }\n]}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/data/mymeds.json b/data/mymeds.json
--- a/data/mymeds.json	(revision 6ea1c9f199a4d1b4ded132336e2a01846a326015)
+++ b/data/mymeds.json	(date 1679087948792)
@@ -6,10 +6,10 @@
             "day": 31
         },
         "drugs": [{
-            "instructions": "eat",
-            "dose": "50g",
+            "instructions": "for diving",
+            "dose": "40g",
             "name": "gillyweed",
-            "indication": "breathing underwater"
+            "indication": "diving"
         }],
         "name": "harry"
     },
@@ -26,5 +26,19 @@
             "indication": "change into someone else"
         }],
         "name": "hermione"
+    },
+    {
+        "birthday": {
+            "month": 11,
+            "year": 2000,
+            "day": 2
+        },
+        "drugs": [{
+            "instructions": "drink",
+            "dose": "50ml",
+            "name": "polyjuice",
+            "indication": "tranform"
+        }],
+        "name": "ron"
     }
 ]}
\ No newline at end of file
